### DISCUSSION TITLE
Change height of bottom tab bar

### DISCUSSION THREAD

**Discussion by scottschindler** [wix/react-native-navigation#7206] [Q&A]
Hey,

Anyone know how to increase the height of the bottom tab bar? I don't see an option anywhere in the docs.

Thank you,
Scott

**Comment by svbutko** [comment]
I might be wrong.

So far the only option I found is using `dimens.xml` on Android (although there are some changes needed to be applied in RNN code to receive correct values from certain methods i.e. height constants). PR and explanation are here #7238

And on iOS it has to be implemented in RNN.

**Reply by svbutko** [reply]
For iOS I found a solution how to change it but it will take some time to apply changes and I don't know yet how everything else inside BottomTabs (e.g. buttons, border, shadow, .etc) will react to this changes.
